front/style/_committee.scss

```diff
@@ -330,6 +330,17 @@
                 }
             }
         }
+
+        &-description {
```

Comment:
@marineb In our private discussion on Slack, you told me you'd like to rename one the CSS classes. Can you you give some details?